lib/runner.js

```diff
@@ -75,12 +77,12 @@ module.exports = (locations = [], formatFn = require('pg-format'), isolationLeve
 
     const tableClause = tableSql(schema, table);
     const updateClause = sqlZip(',', update);
-    const whereClause = R.when(Boolean, R.concat('WHERE '), sqlZip(' AND ', where));
-
-    return _prepare(sqlFormatter, queries, '_update', [tableClause, updateClause, whereClause]);
+    const whereClause = R.when(Boolean, R.concat('WHERE '), sqlZip(' AND ', R.dissoc('_returning', options)));
+    const queryName = conditionalConcat('_update', '_returning', R.equals(options._returning, true));
+    return _prepare(sqlFormatter, queries, queryName, [tableClause, updateClause, whereClause]);
   };
 
-  const _prepareInsert = (queries, schema, table, insertions) => {
+  const _prepareInsert = (queries, schema, table, insertions, options) => {
```

Comment:
I thought this `options` would need to be `options = {} `, but then I saw the line #116 and I realised, that it could be redundant, so I guess it is not needed this time.